lib/hcrypto/rsa-ltm.c

```diff
@@ -548,7 +549,7 @@ ltm_rsa_generate_key(RSA *rsa, int bits, BIGNUM *e, BN_GENCB *cb)
     do {
 	BN_GENCB_call(cb, 2, counter++);
 	CHECK(random_num(&q, bits - bitsp), 0);
-	CHECK(mp_find_prime(&q,128), MP_YES);
+	CHECK(mp_prime_next_prime(&q, trials, 0), MP_OKAY);
```

Comment:
and this one accordingly `CHECK(mp_prime_next_prime(&p, mp_prime_rabin_miller_trials(bits - bitsp), 0), MP_OKAY);`